lib/server/services/lhcPeriod/LhcPeriodStatisticsService.js

```diff
@@ -28,15 +28,28 @@ const { NotFoundError } = require('../../errors/NotFoundError');
  * but this breaks limit clause, so pagination.
  * There is related issue for reference: https://github.com/sequelize/sequelize/issues/11983
  */
-const DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE = [
-    `(SELECT GROUP_CONCAT(DISTINCT(r.lhc_beam_energy))
+const DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE = {
+    query: `(SELECT GROUP_CONCAT(DISTINCT(r.lhc_beam_energy))
         FROM runs AS r
         WHERE r.lhc_period_id = lhcPeriod.id 
             AND r.definition = 'PHYSICS'
```

Comment:
You have a `GROUP_CONCAT` without group by ?

Reply:
yes, because `GROUP_CONCAT` is applied only in scope of subquery, where all records are merged, so there is no need of `CROUP BY` clause